types/lokijs/index.d.ts

```diff
@@ -1292,6 +1292,7 @@ interface CollectionOptions<E> {
     disableMeta: boolean;
     disableChangesApi: boolean;
     disableDeltaChangesApi: boolean;
+    disableFreeze: boolean;
```

Comment:
This property appears to have been added to Lokijs as part of v1.5.12 (see https://github.com/techfort/LokiJS/commit/971473a9a9c50f17fe49f3bf26a1873181dbfc9d). The version number on line 1 of this definition file should be updated to reflect that the definitions are for a newer version of Lokijs.

Reply:
Oh, thanks!
Should I also update the comment on line 8, or is this reserved for feature updates?
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/ce878d43de147cd006746747d3a24aba58f9c376/types/lokijs/index.d.ts#L8

Reply:
Yes, that would be good to update as well!